example/app/app.component.ts

```diff
@@ -34,11 +34,24 @@ import { AppConfig } from './app.config';
     'app.component.scss'
   ],
   template: `
+    <ng-template let-item="item" #referenceTemplate>
```

Comment:
could we even do something simpler? or in the case since this template is too big to embed in ts, you can create `app.component.html` instead.